AlphaWallet/UI/Views/AmountTextField.swift

```diff
@@ -217,12 +227,14 @@ class AmountTextField: UIControl {
     }
 
     private func convertToAlternateAmountNumeric() -> Double? {
-        if let ethToDollarRate = ethToDollarRate, let string = textField.text, let amount = Double(string) {
+        if let ethToDollarRate = cryptoToDollarRate, let string = textField.text, let amount = Double(string) {
             switch currentPair.left {
             case .eth:
                 return amount * ethToDollarRate
             case .usd:
                 return amount / ethToDollarRate
+            case .xDai:
+                return amount
```

Comment:
Better to combine with `.eth`:

```
case .eth, .xDai:
    return amount * ethToDollarRate
```

And probably rename `ethToDollarRate` to `cryptoToDollarRate` too.